build-system/babel-config/import-resolver.js

```diff
@@ -79,8 +81,20 @@ function getImportResolverPlugin() {
   return ['module-resolver', getImportResolver()];
 }
 
+/**
+ * Resolves a filepath using the same logic as the rest of our build pipeline (babel module-resolver).
+ * @param {string} filepath
+ * @return {string}
+ */
+function resolvePath(filepath) {
+  return moduleResolver
```

Comment:
Just curious, what's this actually doing/returning? Can't tell why the slice is there, etc

Reply:
It uses node-style file resolution to return an absolute path to a filepath. I don't know why, but it always returns the path in a relative format `./`, so the `.slice(1)` strips the `.`.

Ideally I'd figure out how to inform `resolvePath` to return a true absolute path, but from cursory research I didn't think that was possible.

Reply:
I guess you could wrap it in `path.resolve()` to get an absolute path, if that would help

Reply:
`path.resolve` vs. `.slice(1)` 🤷. A comment would definitely be helpful here so future readers don't get confused.

Reply:
`path.resolve` is a bit more understandable to read but I think as long as the function comments explain that it returns a relative path I'm fine with w/e

Reply:
From a _file_ in the `/` root path (the second param), the path is `./foo/bar.js`. Why aren't you using `process.cwd()`, where the path would always resolve correctly as relative?

Reply:
As an example, `process.cwd()` as the second arg returns `/amphtml/src/core/context/index.js`

Reply:
The second param is a _file_. If using `process.cwd()` (a _directory_), you need to make it into a file to do resolution from: `path.join(process.cwd(), '[entrypoint]')`

Reply:
Base on your `amphtml` example, it appears it could be any file path, and the second param will be resolved relative to `process.cwd()`. So `amphtml` works fine, because it thinks that's a JS file that it's resolving relative from.

Reply:
Resolving as i've addressed all the points here. Changes in the last batch:
1. Added a comment explaining that the 2nd arg can be any file string
2. Changes the filename to be a nonsensical `anything`